resources/authgear/templates/en/web/authflowv2/forgot_password.html

```diff
@@ -37,12 +37,6 @@ <h1 class="screen-title">{{ template "v2.page.forgot-password.default.title" }}<
             {{ template "v2.page.forgot-password.default.email-input-description" }}
           {{ end }}
         </p>
-        {{ template "authflowv2/__alert_message.html"
-          (dict
-            "Type" "error"
-            "Classname" "mt-4"
-          )
-        }}
```

Comment:
Again, every page should have __alert_message.html, why do you keep wanting to remove it?

Reply:
Oh, the `__alert_message.html` in this file does not do anything right now. Note `Message` param is missing.

As mentioned in DEV-2058

> Currently, all errors is put under input field.

So this commit is just removing unused code

Reply:
I think a proper fix would be to distinguish what error to show in input/ what error to show in alert message. But might need fung input on this?

Reply:
I see.